security/stunnel/src/opnsense/scripts/stunnel/generate_certs.php

```diff
@@ -28,6 +28,8 @@
  */
 
 require_once('plugins.inc');
```

Comment:
Please leave out the certificate generation in this PR, it might have side affects since the CA option should only accept the selected certificates (and no certificates generated from its parents).

The manual (https://www.stunnel.org/static/stunnel.html) suggests that all certificates will be allowed given our current settings.
 
I don't mind an additional PR for adding the chain ca's, but I currently have no time to review and test it.

Reply:
I think we have to talk about usecases here. But for this PR, I agree it is easier to leave out the chain.